new-site/src/docs/elements/design-tokens/colour/components/brand-colour-variation-tokens-table.tsx

```diff
@@ -0,0 +1,67 @@
+  { cssVariable: <code>--color-metro-medium-light</code>, sassVariable: <code>$color-metro-medium-light</code>, hexValue: '#ffcab3', example: <TextAndColorExample color="var(--color-metro-medium-light)" name="Metro medium light variant example" /> },
+  { cssVariable: <code>--color-metro-light</code>, sassVariable: <code>$color-metro-light</code>, hexValue: '#ffeee6', example: <TextAndColorExample color="var(--color-metro-light)" name="Metro light variant example" /> },
+
+  { cssVariable: <code>--color-summer-dark</code>, sassVariable: <code>$color-summer-dark</code>, hexValue: '#cc9200', example: <TextAndColorExample color="var(--color-summer-dark)" name="Summer dark variant example" /> },
+  { cssVariable: <code>--color-summer-medium-light</code>, sassVariable: <code>$color-summer-medium-light</code>, hexValue: '#ffe49c', example: <TextAndColorExample color="var(--color-summer-medium-light)" name="Summer medium light variant example" /> },
+  { cssVariable: <code>--color-summer-light</code>, sassVariable: <code>$color-summer-light</code>, hexValue: '#fff4d4', example: <TextAndColorExample color="var(--color-summer-light)" name="Summer light variant example" /> },
+
+  { cssVariable: <code>--color-suomenlinna-dark</code>, sassVariable: <code>$color-suomenlinna-dark</code>, hexValue: '#e673a5', example: <TextAndColorExample color="var(--color-suomenlinna-dark)" name="Suomenlinna dark variant example" /> },
+  { cssVariable: <code>--color-suomenlinna-medium-light</code>, sassVariable: <code>$color-suomenlinna-medium-light</code>, hexValue: '#ffdbeb', example: <TextAndColorExample color="var(--color-suomenlinna-medium-light)" name="Suomenlinna medium light variant example" /> },
+  { cssVariable: <code>--color-suomenlinna-light</code>, sassVariable: <code>$color-suomenlinna-light</code>, hexValue: '#fff0f7', example: <TextAndColorExample color="var(--color-suomenlinna-light)" name="Suomenlinna light variant example" /> },
+
+  { cssVariable: <code>--color-tram-dark</code>, sassVariable: <code>$color-tram-dark</code>, hexValue: '#006631', example: <TextAndColorExample color="var(--color-tram-dark)" name="Tram dark variant example" /> },
+  { cssVariable: <code>--color-tram-medium-light</code>, sassVariable: <code>$color-tram-medium-light</code>, hexValue: '#a3e3c2', example: <TextAndColorExample color="var(--color-tram-medium-light)" name="Tram medium light variant example" /> },
+  { cssVariable: <code>--color-tram-light</code>, sassVariable: <code>$color-tram-light</code>, hexValue: '#dff7eb', example: <TextAndColorExample color="var(--color-tram-light)" name="Tram light variant example" /> },
+];
+
+export default function BrandColourVariationTokensTable() {
+  return (
+    <Table dense cols={cols} rows={rows} heading="Brand colour variation tokens" headingAriaLevel={3} indexKey="id" renderIndexCol={false} />
```

Comment:
This can be simplified to:

`<Table dense cols={cols} rows={rows} heading="Brand colour tokens" headingAriaLevel={3} indexKey="cssVariable" />`